### PR TITLE
Update build to publish from Travis

### PR DESCRIPTION
**Pull request by ensonik**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,12 @@
 language: scala
 scala:
-   - 2.10.4
-   - 2.11.2
+- 2.10.4
```
Comment (jletroui) [inline .travis.yml]: Any reason it is not the same 2.10.5 and 2.11.6 versions as in the build.sbt file?

Comment (ensonik) [inline .travis.yml]: I hadn't seen it. I'll update.

**Comment by ensonik** [comment]
@jletroui The build just passed, but I didn't do anything to the code ... Still a bit flaky.

**Comment by ensonik** [comment]
@jletroui  I lied: Travis was saying "Green" all over, but it was lying to me. It turns out that the scala 2.10.5 version is still failing.

**Pull request merged**